docs/sphinx/developers/generating-test-images.txt

```diff
@@ -39,66 +39,109 @@ global metadata map:
     echo "my.key=some.value" >> my-special-test-file.fake.ini
 
 
-There are a few other keys that can be added as well:
+There are several other keys that can be added as well, these are listed below
+with their default values.
 
 .. tabularcolumns:: |p{3cm}|p{11cm}|
 
 .. list-table::
     :header-rows: 1
-    :widths: 30, 70
+    :widths: 30, 70, 10
```

Comment:
Thanks both. I was convinced I had reduced one of the other widths! I'll fix both.